tools/pipelines/repo-policy-check.yml

```diff
@@ -15,52 +15,83 @@ pr:
 - lts
 - release/*
 
-pool:
-  Small
-
 variables:
 - name: skipComponentGovernanceDetection
   value: true
 - name: pnpmStorePath
   value: $(Pipeline.Workspace)/.pnpm-store
 
-steps:
-- template: templates/include-use-node-version.yml
-
-- template: templates/include-install-pnpm.yml
+# The `resources` specify the location and version of the 1ES PT.
+resources:
+  repositories:
+  - repository: m365Pipelines
+    type: git
+    name: 1ESPipelineTemplates/M365GPT
+    ref: refs/tags/release
+extends:
+  # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
```

Comment:
Nit (for future changes): It would be good to define "PT" and "SDL" somewhere in documentation that can be referred to easily.